lib/configuration.yaml

```diff
@@ -135,6 +135,21 @@ dev: &dev
         commonName: "Daedalus Wallet"
         expiryDays: 365
 
+  wallet: &dev_wallet
+    # Configuration options for throttling the API. Items at the top level
+    # apply as a default setting to all requests. A 'null' value indicates that
+    # throttling is disabled.  To enable throttling, change the null value for
+    # the keys that are commented out below.
+    throttle: null
+    # throttle:
+    # # Number of requests per throttling period.
+    #   rate: 200
+    #   # Amount of microseconds in a throttling period.
+    #   period: 1_000_000
```

Comment:
https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Wallet/Web/Methods/Payment.hs#L80 note that we have "throttled" 6 seconds here - which is 6 times more then here. I know new data layer is probably more optimized then the old one - but maybe we should measure or check why did we pick 6 in v0. I know it was due to sev1 performance problem

Reply:
Note that this is actually quite a bit faster -- we have `rate` requests per `period` microseconds, so 200 requests permitted per second in this configuration, across the board. It is disabled entirely by default.

More fine-grained throttling can easily be done at the handler level, I think.

Reply:
> so 200 requests permitted per second in this configuration

yes - we just have to check eventually is our wallet layer up to the task. I am not sure have we run benchmarks on new wallet layer and compared it to the old one